raiden-ts/src/config.ts

```diff
@@ -99,7 +102,9 @@ export function makeDefaultConfig(
       [Capabilities.NO_DELIVERY]: true,
       [Capabilities.NO_RECEIVE]: true,
       [Capabilities.NO_MEDIATE]: true,
+      [Capabilities.WEBRTC]: true,
     },
+    fallbackIceServers: [{ urls: 'stun:stun.l.google.com:19302' }],
```

Comment:
Do we need to mention this in the privacy policy or get some kind of allowance?

Reply:
Thanks :) I am picking this up

Reply:
I don't think this is needed, these are public servers, and no information pass through them, it's used only as "connection faciliators", with a role not that different from e.g. DNS, and I don't remember we getting an allowance for that.
Nonetheless, there're tons of those public servers available, we can switch to some more respectable provider, and also this will be mostly unused once our matrix servers provide TURN, which I already implemented here, and our own TURN servers are preferred over these fallback STUN servers.

Reply:
Yes I just asked because @kelsos mentioned getting approval for hosting on GH pages was quite the hassle and that we should not deploy the dApp somewhere else (e.g. Netlify)
If this is a problem (not sure if it is) then switching to another server wouldn't solve the issue as we would still need users to inform that this could happen in the policy